zetaclient/bitcoin/bitcoin_client.go

```diff
@@ -1233,36 +1252,38 @@ func (ob *BTCChainClient) checkTssOutTxResult(cctx *types.CrossChainTx, hash *ch
 
 	// differentiate between normal and restricted cctx
 	if compliance.IsCctxRestricted(cctx) {
-		err = ob.checkTSSVoutCancelled(params, rawResult.Vout)
+		err = ob.checkTSSVoutCancelled(params, rawResult.Vout, ob.chain)
 		if err != nil {
 			return errors.Wrapf(err, "checkTssOutTxResult: invalid TSS Vout in cancelled outTx %s nonce %d", hash, nonce)
 		}
 	} else {
-		err = ob.checkTSSVout(params, rawResult.Vout)
+		err = ob.checkTSSVout(params, rawResult.Vout, ob.chain)
```

Comment:
Why do you need to pass the ob.chain if the function is a member of "ob" already? Same with other functions above.

Reply:
https://github.com/zeta-chain/node/pull/1942/commits/f02cc5253a5fbc6f0402c41ad0905837de3eb2c9